articles/sso/current/index.md

```diff
@@ -71,8 +71,7 @@ For more detailed information on how to implement this, please refer to the foll
 * [Client-Side SSO (Single Page Apps)](/sso/current/single-page-apps-sso)
```

Comment:
You need to also update the text before the sample, since it still describes the old sample (uses the `getSSOData()` function, which is legacy)